packages/commercetools/api-client/src/types/Api.ts

```diff
@@ -7,9 +7,21 @@ export interface CustomQuery<T> {
   variables: T;
 }
 
-type CustomQueryFn<T = any> = (query?: any, variables?: T) => CustomQuery<T>
+export type CustomQueryFn<T = any> = (query?: any, variables?: T) => CustomQuery<T>;
+export type CartCustomQueryFn<T> = (
+  cart?: CustomQuery<T>,
+  user?: CustomQuery<T>
+) => { cart: CustomQuery<T>; user: CustomQuery<T> };
 
-export const getCustomQuery = <T = any>(customQueryFn: CustomQueryFn<T>, defaultQuery) => customQueryFn ? customQueryFn() : { query: defaultQuery, variables: {} };
+export const getCustomQuery = <T = any>(customQueryFn: CustomQueryFn<T>, defaultQuery) =>
+  customQueryFn ? customQueryFn() : { query: defaultQuery, variables: {} };
+export const getCartCustomQuery = <T = any>(customQueryFn: CartCustomQueryFn<T>, customQueries) =>
+  customQueryFn
+    ? customQueryFn()
+    : {
+      cart: customQueries.cart ? { query: customQueries.cart.query, variables: {} } : null,
```

Comment:
I remember we talked about naming for this example to be more descriptive, are there any other examples that need 2 queries/mutations?

Reply:
@filrak Not yet.